### DISCUSSION TITLE
золо оформление

### DISCUSSION THREAD

**Discussion by FatherOfLife** [FatherOfLife/vickTorina#2] [General]
1

**Comment by FatherOfLife** [comment]
[#include](tg://search_hashtag?hashtag=include) <iostream>

int main() {
    int rows;
    std::cout << "Enter the number of students: ";
    std::cin >> rows;

    int** arr = new int* [rows];
    int* cols = new int[rows];

    for (int i = 0; i < rows; i++) {
        std::cout << "Enter the number of grades for student " << i << ": ";
        std::cin >> cols[i];
        arr[i] = new int[cols[i]];
        for (int j = 0; j < cols[i]; j++) {
            int grade;
            std::cout << "Enter grade " << j + 1 << " for student " << i << ": ";
            std::cin >> grade;
            arr[i][j] = grade;
        }
    }
    for (int i = 0; i < rows; i++) {
        std::cout << "Student " << i << ": ";
        for (int j = 0; j < cols[i]; j++) {
            std::cout << arr[i][j] << " ";
        }
        std::cout << std::endl;
    }
    for (int i = 0; i < rows; i++) {
        delete[] arr[i];
    }
    delete[] arr;
    delete[] cols;

    return 0;
}

**Reply by zolosquad** [reply]
___-МЕНЮ-___ 
 -ВОПРОС №1-
*/ВАШ ВЫБОР\*

**Reply by zolosquad** [reply]
___-МЕНЮ-___ 
 -ВОПРОС №1-
*/ВАШ ВЫБОР\*

**Reply by zolosquad** [reply]
\
